ts/src/test/static/data/bingx.json

```diff
@@ -4,18 +4,120 @@
     "outputType": "urlencoded",
     "methods": {
         "createOrder": [
-            {
```

Comment:
now tests are failing not sure if it's related to the fact we're now explicitly using safeFloat

Reply:
Yeah basically the issue is that parseFloat converts everything to floats, even the integers so that's a problem in python because 5 is different from 5.0 whereas they're the exact same thing in JS